test/test.auth.ts

```diff
@@ -210,6 +206,12 @@ describe('OAuth2 client', () => {
     };
 
     nock(Utils.baseUrl).get('/drive/v2/files/wat').reply(200);
+    let scope = nock(Utils.baseUrl).post('/oauth2/v4/token').reply(200, {
```

Comment:
Sorry for missing this earlier.  You can just use `.times(s)` and a `const` here to ensure it runs twice :)

Reply:
After some looking at the code, I guess the point is that it should **not** run at all. That's what happens in node8 and node9, and what does not happen in node4 and node6.